indico/core/db/sqlalchemy/custom/int_enum.py

```diff
@@ -72,11 +73,9 @@ def coerce_set_value(self, value):
             return None
         return self.enum(value)
 
-    def _set_table(self, column, table):
-        e = CheckConstraint(type_coerce(column, self).in_(x.value for x in self.enum if x not in self.exclude_values),
-                            'valid_enum_{}'.format(column.name))
-        e.info['alembic_dont_render'] = True
-        assert e.table is table
+    def _set_parent_with_dispatch(self, parent):
```

Comment:
https://bitbucket.org/zzzeek/sqlalchemy/issues/3832/typedecorator-schematype-combination-not#comment-31530065